From b95c288c7d897ed3ed9268572b7c097ed2652907 Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@oracle.com>
Date: Fri, 8 Apr 2011 13:24:28 -0700
Subject: [PATCH:libX11] Install xml versions of specs even if HAVE_XMLTO is false

Moves HAVE_XMLTO check into docbook.am, more closely matches behaviour
from before docbook.am changes (commit e8c76407d2f6e3)

Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
---
 docbook.am                       |    2 ++
 specs/XIM/Makefile.am            |    2 --
 specs/XKB/Makefile.am            |    2 --
 specs/i18n/framework/Makefile.am |    2 --
 specs/i18n/localedb/Makefile.am  |    2 --
 specs/i18n/trans/Makefile.am     |    2 --
 specs/libX11/Makefile.am         |    2 --
 7 files changed, 2 insertions(+), 12 deletions(-)

diff --git a/docbook.am b/docbook.am
index 3cf21e8..6937083 100644
--- a/docbook.am
+++ b/docbook.am
@@ -20,6 +20,7 @@ shelf_DATA =
 # DocBook/XML file with chapters, appendix and images it includes
 dist_shelf_DATA = $(docbook) $(chapters)
 
+if HAVE_XMLTO
 #
 # Generate DocBook/XML output formats with or without stylesheets
 #
@@ -91,5 +92,6 @@ XSLTPROC_FLAGS =					\
 
 endif HAVE_XSLTPROC
 endif HAVE_STYLESHEETS
+endif HAVE_XMLTO
 
 CLEANFILES = $(shelf_DATA) $(sgmldbs_DATA)
diff --git a/specs/XIM/Makefile.am b/specs/XIM/Makefile.am
index a1fa72d..68b0b3f 100644
--- a/specs/XIM/Makefile.am
+++ b/specs/XIM/Makefile.am
@@ -1,6 +1,5 @@
 
 if ENABLE_SPECS
-if HAVE_XMLTO
 
 # Main DocBook/XML files (DOCTYPE book)
 docbook = xim.xml
@@ -21,5 +20,4 @@ shelfdir = $(docdir)/XIM
 # Generate DocBook/XML output formats with or without stylesheets
 include $(top_srcdir)/docbook.am
 
-endif HAVE_XMLTO
 endif ENABLE_SPECS
diff --git a/specs/XKB/Makefile.am b/specs/XKB/Makefile.am
index 3939db6..90771cc 100644
--- a/specs/XKB/Makefile.am
+++ b/specs/XKB/Makefile.am
@@ -1,6 +1,5 @@
 
 if ENABLE_SPECS
-if HAVE_XMLTO
 
 # Main DocBook/XML files (DOCTYPE book)
 docbook = xkblib.xml
@@ -59,5 +58,4 @@ shelfdir = $(docdir)/XKB
 # Generate DocBook/XML output formats with or without stylesheets
 include $(top_srcdir)/docbook.am
 
-endif HAVE_XMLTO
 endif ENABLE_SPECS
diff --git a/specs/i18n/framework/Makefile.am b/specs/i18n/framework/Makefile.am
index 6e8b010..350b7df 100644
--- a/specs/i18n/framework/Makefile.am
+++ b/specs/i18n/framework/Makefile.am
@@ -1,6 +1,5 @@
 
 if ENABLE_SPECS
-if HAVE_XMLTO
 
 # Main DocBook/XML files (DOCTYPE book)
 docbook = framework.xml
@@ -14,5 +13,4 @@ shelfdir = $(docdir)/framework
 # Generate DocBook/XML output formats with or without stylesheets
 include $(top_srcdir)/docbook.am
 
-endif HAVE_XMLTO
 endif ENABLE_SPECS
diff --git a/specs/i18n/localedb/Makefile.am b/specs/i18n/localedb/Makefile.am
index 8db6a7b..0c52a13 100644
--- a/specs/i18n/localedb/Makefile.am
+++ b/specs/i18n/localedb/Makefile.am
@@ -1,6 +1,5 @@
 
 if ENABLE_SPECS
-if HAVE_XMLTO
 
 # Main DocBook/XML files (DOCTYPE book)
 docbook = localedb.xml
@@ -11,5 +10,4 @@ shelfdir = $(docdir)/localedb
 # Generate DocBook/XML output formats with or without stylesheets
 include $(top_srcdir)/docbook.am
 
-endif HAVE_XMLTO
 endif ENABLE_SPECS
diff --git a/specs/i18n/trans/Makefile.am b/specs/i18n/trans/Makefile.am
index cdec750..6e4baf1 100644
--- a/specs/i18n/trans/Makefile.am
+++ b/specs/i18n/trans/Makefile.am
@@ -1,6 +1,5 @@
 
 if ENABLE_SPECS
-if HAVE_XMLTO
 
 # Main DocBook/XML files (DOCTYPE book)
 docbook = trans.xml
@@ -11,5 +10,4 @@ shelfdir = $(docdir)/trans
 # Generate DocBook/XML output formats with or without stylesheets
 include $(top_srcdir)/docbook.am
 
-endif HAVE_XMLTO
 endif ENABLE_SPECS
diff --git a/specs/libX11/Makefile.am b/specs/libX11/Makefile.am
index 0e59ad8..c4de67e 100644
--- a/specs/libX11/Makefile.am
+++ b/specs/libX11/Makefile.am
@@ -1,6 +1,5 @@
 
 if ENABLE_SPECS
-if HAVE_XMLTO
 
 # Main DocBook/XML files (DOCTYPE book)
 docbook = libX11.xml
@@ -36,5 +35,4 @@ shelfdir = $(docdir)
 # Generate DocBook/XML output formats with or without stylesheets
 include $(top_srcdir)/docbook.am
 
-endif HAVE_XMLTO
 endif ENABLE_SPECS
-- 
1.7.3.2

